hc/settings.py

```diff
@@ -153,3 +161,7 @@
     from .local_settings import *
 else:
     warnings.warn("local_settings.py not found, using defaults")
+
+
+STATIC_ROOT = os.path.join(BASE_DIR, 'staticfiles')
+STATIC_URL = '/static/'
```

Comment:
This breaks Travis, I had it before and Travis failed should I add it to work only on Heroku platform?